middleware/replacer.go

```diff
@@ -40,6 +41,7 @@ func NewReplacer(r *http.Request, rr *responseRecorder, emptyValue string) Repla
 			"{host}":     r.Host,
 			"{path}":     r.URL.Path,
 			"{query}":    r.URL.RawQuery,
+			"{frag}":     r.URL.Fragment,
```

Comment:
Cool then. I'll update this.

Remember to update the docs for `rewrite` in the next release.